### PR TITLE
Portfolio: Desktop Version

### PR DESCRIPTION
**Pull request by clin2on3mun**
**This is the Desktop version of the portfolio it has the following files:**

- index.html file that has HTML Semantics and contents(eg: About me, Contact form).
- styles.css  file that applies the designs and styles to the Html files 
       1.   grid layout
       2. flexbox

- Images directory that stores images

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -5,439 +5,1094 @@
...
+    height: 5px;
+    background-color: #0d151e;
+    margin: 3px auto;
+    border-radius: 10px;
+  }
+
+  .footer-hr {
+    padding-top: 10px;
+  }
 }
```
Comment (zieeco) [inline styles.css:496]: ### MOBILE VERSION


- Hey, please, the mobile version is having a horizontal scroll effect which is not a great experience for the user. please, let's fix this to improve the user experience of your portfolio website.

File: styles.css [hunk 2]
```diff
@@ -5,439 +5,1094 @@
...
+    grid-template-columns: 320px 320px 320px;
+    grid-column-gap: 120px;
+    margin-top: 100px;
+    margin-left: 100px;
+  }
+
+  .experience {
+    display: flex;
+  }
+
```
Comment (zieeco) [inline styles.css]: - Please, let's align the experience section in the center to be close to the Figma design below:
- Let's use the correct font-family in the list items as you've used another font that is not the same as the one used. See the image below:

*your design*
Screenshot from 2023-01-27 12-50-00


*Figma design*
Screenshot from 2023-01-27 12-49-08

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @clin2on3mun ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A



**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @clin2on3mun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**